plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt

```diff
@@ -190,7 +192,7 @@ open class CargoBuildTask : DefaultTask() {
                     environment("RUST_ANDROID_GRADLE_LINKER_WRAPPER_PY",
                             File(project.rootProject.buildDir, "linker-wrapper/linker-wrapper.py").path)
                     environment("RUST_ANDROID_GRADLE_CC", cc)
-                    environment("RUST_ANDROID_GRADLE_CC_LINK_ARG", "-Wl,-soname,lib${cargoExtension.libname!!}.so")
+                    environment("RUST_ANDROID_GRADLE_CC_LINK_ARG", "-o,$cargoOutputDir/lib${cargoExtension.libname!!}.so")
```

Comment:
Hmm, I wonder if we should be still setting the soname in addition to this. Do you know if the current version still ends up with the correct soname?

Reply:
I am not sure.